### PR TITLE
Second set of changes for RAIL

### PR DESCRIPTION
**Pull request by joezuntz**

### CODE REVIEW

File: ceci/stage.py [hunk 1]
```diff
@@ -160,20 +203,19 @@ def load_configs(self, args):
...
         # We prefer to receive explicit filenames for the outputs but will
         # tolerate missing output filenames and will default to tag name in
         # current folder (this is for CWL compliance)
         self._outputs = {}
         for i, x in enumerate(self.output_tags()):
+            aliased_tag = self.get_aliased_tag(x)
             if args.get(x) is None:
                 ftype = self.outputs[i][1]  #pylint: disable=no-member
-                self._outputs[self.get_aliased_tag(x)] = ftype.make_name(x)
+                aliased_tag = self.get_aliased_tag(x)
```
Comment (joezuntz) [inline ceci/stage.py:214]: Is this a repeat of the line above?

**Review by joezuntz — COMMENTED** [review]
It won't let me approve, but once the CI is fixed then this looks good (I'll test with TXPipe quickly also).

**Comment by joezuntz** [comment]
Running on TXPipe, I notice that --name seems to always be set by ceci when it makes its command line. I think that wasn't intended? WIll try to find the right bit.  

Otherwise the TX pipeline seems to work with this version.

Could you bump the version number also?  I need to do a release afterwards for pip.

File: ceci/stage.py [hunk 2]
```diff
@@ -1113,6 +1170,9 @@ def generate_command(cls, inputs, config, outputs, aliases=None):
                 raise ValueError(f"Missing input location {aliased_tag} {str(inputs)}") from msg
             flags.append(f"--{tag}={fpath}")
 
+        if instance_name is not None:
```
Comment (joezuntz) [inline ceci/stage.py:1173]: I think this should only happen when instance_name is not the same as class name?

Comment (eacharles) [reply ceci/stage.py:1173]: Or instance_name is not the same as cls.name (Not quite the same thing).

File: ceci/stage.py (hunk 1 above)
Comment (eacharles) [reply ceci/stage.py:214]: It is, good catch.

**Pull request merged**